blocks/init/src/Blocks/components/icon/docs/story.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { Fragment } from '@wordpress/element';
+import readme from './readme.md';
```

Comment:
you must include mdx file. How is this working in Storybook? have you tested it?

Reply:
Fixed the include and tested in storybook. All seems good now.